Waypoint doc updates (#15051)
istio/istio.io: content/en/docs/ambient/usage/l7-features/index.md

---
title: Use Layer 7 features
description: Supported features when using a L7 waypoint proxy.
weight: 50
owner: istio/wg-networking-maintainers
test: no
---

With a waypoint proxy deployed for a workload, you can enable more of [Istio's features](/docs/concepts).

Ambient mode supports configuring waypoints using the Kubernetes Gateway API. Configurations that apply to a Gateway API are called _policies_.

{{< warning >}}

> The Istio traffic management APIs (virtual service, destination rules etc) are not supported by the ambient data plane mode.

Reviewer: ```suggestion
The Istio classic traffic management APIs (virtual service, destination rules etc) remains at alpha for the ambient data plane mode.
```

Reviewer: Maybe just add them to the table below?

Author: We would then have to call out the attachment type being a selector as opposed to a targetRef.  I'd love this to be in the documentation but it's too complicated to add in this PR imo.

{{< /warning >}}

## Traffic routing

With a waypoint proxy deployed, you can use the following API types:

|  Name  | Feature Status | Attachment |
| --- | --- | --- |
| `HTTPRoute` | Beta | `parentRefs` |
| `TCPRoute` | Beta | `parentRefs` |

> | `TLSRoute` | Beta | `parentRefs` |

Reviewer: +1 they should be marked as Alpha.  Not sure if we need a tip - i basically hate too many tips in docs :)

|  Name  | Feature Status | Attachment |
| --- | --- | --- |
| `HTTPRoute` | Beta | `parentRefs` |
| `TCPRoute` | Alpha | `parentRefs` |
| `TLSRoute` | Alpha | `parentRefs` |

> {{< tip >}}
> `TCPRoute` and `TLSRoute` still have Alpha status in Kubernetes Gateway API. Caution is advised if considering their adoption.
> {{< /tip >}}

Reviewer 2: optional, I think Lin preferred no tip so remove if Lin or yourself feel strongly about it

```suggestion
```
